gk/fib.c

```diff
@@ -1623,3 +1624,286 @@ del_fib_entry(const char *ip_prefix, struct gk_config *gk_conf)
 
 	return del_fib_entry_numerical(&prefix_info, gk_conf);
 }
+
+static int
+fillup_gk_fib_ether_entry(struct ether_cache *eth_cache,
+	struct gk_fib_ether_entry *entry)
+{
+	entry->stale = eth_cache->stale;
+	rte_memcpy(&entry->nexthop_ip, &eth_cache->ip_addr,
+		sizeof(entry->nexthop_ip));
+	rte_memcpy(&entry->l2_hdr, &eth_cache->l2_hdr, sizeof(entry->l2_hdr));
+	entry->ref_cnt = rte_atomic32_read(&eth_cache->ref_cnt);
+
+	return 0;
```

Comment:
Since this can't return an error, I think we can make this function `void` and simplify callers to not have to check the return value.